### PR TITLE
Respect COMPOSE_FILE envvar

### PR DESCRIPTION
**Pull request by soulshake**
Respect `COMPOSE_FILE` environment variable in `convox start`, `convox doctor`, `convox build` and `convox deploy`.

Addresses #1626.

### CODE REVIEW

File: cmd/convox/doctor.go [hunk 1]
```diff
@@ -149,23 +149,24 @@ func cmdDoctor(c *cli.Context) error {
 	}
 
 	stdcli.Writef("\n\n### Build: Service\n")
-	startCheck("<file>docker-compose.yml</file> found")
-	_, err := os.Stat("docker-compose.yml")
+	dcm := helpers.DetectComposeFile()
```
Comment (MiguelMoll) [inline cmd/convox/doctor.go]: This is nitpicking, but perhaps we could have one `dcm = helpers.DetectComposeFile()` variable up in the `var()` scope.

**Comment by soulshake** [comment]
[Respect COMPOSE_FILE envvar](https://trello.com/c/sNt3fuDs/1807-respect-compose-file-envvar)

**Comment by idris** [comment]
@soulshake will this work if you set `COMPOSE_FILE` in `.env`?

**Comment by soulshake** [comment]
@idris No, `.env` is only applied to the containers' environment.

**Pull request closed without merging**